src/main/java/models/Product.java

```diff
@@ -8,12 +8,13 @@ public class Product {
 	private String brand;
 	private int quantity;
 	private double price;
+	private int amount;
```

Comment:
Is 'amount' in Product being used? if so, you should add it to the SQL schema and seed data files.

Also if it is being used, won't this result in problems from multiple carts having the same Products? Ex. if person A and person B have 1 of product A in their carts, if person A increases the quantity to 2, will it increase in person B's cart as well?

Reply:
It's used in the cart so we need it but because the cart is client side it shouldn't create that kind of conflict

Reply:
Ok, I guess its not really a concern anyway if only the TA is gonna be grading it

Reply:
Resolving as said TA has access to these comments 😅